flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateCheckpointWriterTest.java

```diff
@@ -95,12 +118,12 @@ public void flush() throws IOException {
 
 		FlushRecorder dataStream = new FlushRecorder();
 		final ChannelStateCheckpointWriter writer = new ChannelStateCheckpointWriter(
-			1L,
-			new ChannelStateWriteResult(),
-			new ChannelStateSerializerImpl(),
-			NO_OP_RUNNABLE,
-			new MemoryCheckpointOutputStream(42),
-			dataStream
+				1L,
```

Comment:
nit: unnecessary formatting changes, I guess it was done automatically by IDE.

Reply:
Yes, accidental change, reverted it.